src/config/auth.config.ts

```diff
@@ -332,9 +344,13 @@ export default registerAs('authConfig', () => {
 
   const authConfig = authSchema.validate(
     {
-      email: {
-        enableLogin: process.env.HD_AUTH_EMAIL_ENABLE_LOGIN,
-        enableRegister: process.env.HD_AUTH_EMAIL_ENABLE_REGISTER,
+      session: {
+        secret: process.env.HD_SESSION_SECRET,
+        lifeTime: parseOptionalInt(process.env.HD_SESSION_LIFE_TIME),
```

Comment:
I think the english word lifetime would be fine without camelcasing it.

Reply:
Then it should be 
```
lifetime: parseOptionalInt(process.env.HD_SESSION_LIFETIME),
```
correct?

Reply:
Yes

Reply:
done